src/components/error-dart.js

```diff
@@ -1,11 +1,71 @@
 import React from 'react'
+import axios from 'axios'
 import { Header, Button } from 'semantic-ui-react'
 
-import { getTheme, urlErrorAssets } from 'formula_one'
+import { getTheme, urlErrorAssets, urlWhoAmI } from 'formula_one'
 
 import blocks from '../css/error-dart.css'
+import { element } from 'prop-types'
 
 export default class ErrorDart extends React.PureComponent {
+  state = {
+    isFacultyOrGuest: false
+  }
+
+  isFacultyOrGuest = (roles) => {
+    let isFacultyOrGuest = false
+    roles.forEach(element => {
+      if (element.role === 'FacultyMember' || element.role === 'Guest') {
```

Comment:
@pradumangoyal I believe there would be a constant or some better way to do this. :sweat_smile: 
If we hardcode it everywhere, then I think this poses no redundancy.